.github/workflows/main.yaml

```diff
@@ -11,37 +11,41 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
I wonder if tests are necessary to be run on two platforms.

Reply:
I think so because we want to ensure portability among these two platforms.